src/libnmstate/schemas/interfaces.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
I agree, I think it will go in that direction.
For now, it is hard to "see" what is the most suitable way to split it. We need some mass to see pros/cons.
Lets do it in steps.
Importing/Referencing it is supposed to be very easy.